home.css

```diff
@@ -0,0 +1,674 @@
+@font-face {
+  font-family: cocogoose;
+  src: url(fonts/Cocogoose-font.otf);
+}
+
+html,
+body {
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  font-size: 16px;
+  font-family: "lato", sans-serif;
+}
+
+.event-details {
+  padding-bottom: 10%;
+}
+
+p {
+  line-height: 1.1rem;
+}
+
+.menu-content,
+.top-bar-section,
+.close-menu,
+.logo-desktop a {
+  display: none;
+}
+
+.top-bar-section {
+  background-color: #3e3e3e;
+}
+
+.navigation-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  width: 112%;
+}
+
+.nav-toggle {
+  position: absolute;
+  top: 1rem;
+  cursor: pointer;
+  left: 1rem;
+}
+
+.close-menu {
+  cursor: pointer;
+  z-index: 10;
+}
+
+.close-menu p {
+  font-size: 3.2rem;
+  position: absolute;
+  margin: 0;
+  top: 1.3rem;
+  left: 1.3rem;
+  color: black;
+  z-index: 100;
+}
+
+.logo-desktop a {
+  position: absolute;
+  top: 1.3rem;
+  right: 1.5rem;
+  text-decoration: none;
+  color: black;
+}
```

Comment:
- [ ] the `logo` and the menu in the `nav` is not displaying correctly in the mobile version
![log](https://user-images.githubusercontent.com/61562626/129460354-47276045-1eb2-463a-a4d8-e7be823bc58d.jpeg)

Reply:
This issue is well resolved ✔️